custom-connectors/secplugs/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
```

Comment:
@Ajaatsecplugs The connectionParameters property has mentioned with empty body in apiProiperties.json file. Empty connection object will be created automatically without any input from an user. It is less secure to use empty connection parameters. Please update with the right definition or let us know the reason if you have chosen for "no authentication".

Reply:
we have chosen no for authentication.